examples/step-50/doc/results.dox

```diff
@@ -218,7 +247,21 @@ you can compare error rates.
 
 <h4> Coarse solver </h4>
 
-A more interesting example would involve a complicated coarse mesh (see
-step-49 for inspiration). This then requires a more sophisticated coarse
-solver for the geometric multigrid methods. A common approach here is a switch
-to AMG by assembling the coarse matrix (even for the matrix-free version).
+A more interesting example would involve a more complicated coarse mesh (see
+step-49 for inspiration). The issue in that case is that the coarsest
+level of the mesh hierarchy is actually quite large, and one would
+have to think about ways to solve the coarse level problem
+efficiently. (This is not an issue for algebraic multigrid methods
+because they would just continue to build coarser and coarser levels
+of the matrix, regardless of their geometric origin.)
+
+In the program here, we simply solve the coarse level problem with a
+Conjugate Gradient method without any preconditioner. That is acceptable
+if the coarse problem is really small -- here it is a $9\times 9$
+matrix in 2d, and a $27\times 27$ matrix in 3d. But if the coarse mesh
```

Comment:
These are the numbers for a single cell, right? Shouldn't it be 117 in 2d and something around a few hundreds in 3d since we have an L-shaped domain? @tcclevenger can you please check?

Reply:
Correct, it's 3 cells in 2d and 7 cells in 3d. The number of DoFs are 21 in 2d and 117 in 3d.

Reply:
Ah, fair point. I'll update.